opentech/apply/funds/views.py

```diff
@@ -125,6 +138,48 @@ class SubmissionDetailView(ViewDispatcher):
     applicant_view = ApplicantSubmissionDetailView
 
 
+@method_decorator(login_required, name='dispatch')
+class SubmissionEditView(UpdateView):
+    """
+    Converts the data held on the submission into an editable format and knows how to save
+    that back to the object. Shortcuts the normal update view save approach
+    """
+    model = ApplicationSubmission
+
+    def dispatch(self, request, *args, **kwargs):
+        if request.user != self.get_object().user:
```

Comment:
Admins should be able to edit the submission, imho

Reply:
There is another ticket that will cover editing to a greater extent. I want to cover those needs there. This treads on the toes of that ticket. But the requirement of submitting a a proposal when requested is what drove me to including the basic editing part in this.